testing/qgb.go

```diff
@@ -13,3 +21,19 @@ func NewEVMClient(key *ecdsa.PrivateKey) *evm.Client {
 	// the simulated backend doesn't provide an RPC endpoint.
 	return evm.NewClient(logger, nil, key, "", 100000000)
 }
+
+func NewOrchestrator(
+	node *TestNode,
+) *orchestrator.Orchestrator {
```

Comment:
[non blocking nit]
why format this like this?

Reply:
no reason :D just happened to be like that.  should I change?

Reply:
@evan-forbes Can I merge or I should make the change?

Reply:
up to you! if we don't have linters for it, then I don't think its worth blocking on.

Reply:
Cool, thanks